charts/enterprise-metrics/values.yaml

```diff
@@ -6,9 +6,9 @@
 
 # Container image settings.
 # Since the image is unique for all microservices, so are image settings.
-image:
+image: &image
   repository: grafana/metrics-enterprise
-  tag: v1.7.0
+  tag: r176-00920039
```

Comment:
I'm sure you're away, but just flagging to make sure that we update this on release.

Reply:
yes, just got RC